### ISSUE TITLE
Term Request: Hutchinson sign of herpes zoster

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11757]
**Preferred term label:**
Hutchinson sign of herpes zoster

**Synonyms:**
* Hutchinson's sign (zoster)
* Hutchinson sign (VZV)
* Hutchinson's sign of herpes zoster ophthalmicus
* Nasociliary nerve involvement in herpes zoster
* Herpetic vesicle of nasal tip

**Definition (free text):**
The occurrence of vesicular or erythematous skin eruptions localized to the tip, side, or ala of the nose during an episode of herpes zoster. This finding indicates involvement of the external nasal branch of the nasociliary nerve (ophthalmic division of the trigeminal nerve, cranial nerve V1) and serves as a strong clinical predictor of ocular involvement (herpes zoster ophthalmicus).

**Parent term (use hpo.jax.org/app):**
Skin vesicle (HP:0200037)

**Your nano-attribution (ORCID):**
0000-0001-9042-0508
